### ISSUE TITLE
A new line is created when characters are entered and then removed with the backspace key

### ISSUE THREAD

**Issue by pomartel** [froala/wysiwyg-editor#3655]
##### Expected behavior.
When you type text in the editor then hit backspace to remove it, it should not create a new line (`<br>`).

##### Actual behavior.
It creates a new line (`<br>`) and the text editor height gets expanded.

##### Steps to reproduce the problem.
https://jsfiddle.net/oLh5t8a7/
Type a letter in the editor then press backspace. A new line is created.
This only happens when the `enter` option is set to `FroalaEditor.ENTER_BR`

##### Editor version.
The bug happens in versions 2 and 3.

##### OS.
macOS Mojave

##### Browser.
Firefox 68 (does not happen in Chrome)

**Comment by casualuser** [comment]
Hello, 

We have tried to catch the sequence of required actions for issue reproducing but can't repeat it.
http://recordit.co/Z6i3tgwARs

Please provide a screen record and more details about required steps and conditions for this issue.

**Comment by pomartel** [comment]
Here are the steps to reproduce:

**Type a SINGLE letter in the editor then press backspace. A new line is created.**

In your video, you never backspace all the way back, that's why the new line is not created. Just type a single letter and then backspace. Make sure you are using Firefox: https://jsfiddle.net/oLh5t8a7/

**Comment by alexanderduke** [comment]
Can confirm issue is still present in version 4.0.5

**Comment by joseix6** [comment]
yeah sill happening on version 4.0.13, this specific issue -> https://github.com/froala/wysiwyg-editor/issues/3655, on chrome, in FF is working fine.

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
